projects/packages/masterbar/src/nudges/additional-css/class-atomic-additional-css-manager.php

```diff
@@ -54,20 +54,22 @@ public function register_nudge( \WP_Customize_Manager $wp_customize_manager ) {
 	}
 
 	/**
-	 * Get the plan name.
+	 * Get the plan.
 	 *
 	 * @return mixed
 	 */
-	protected function get_plan_name() {
-		return \Automattic\Jetpack\Plans::get_plan( 'business_bundle' )->product_name_short;
+	protected function get_plan() {
+		$plan_slug = apply_filters( 'wpcom_customize_css_plan_slug', 'value_bundle' );
```

Comment:
Do we use this filter on Atomic? Also, this means users not part of the summer special will get the upsell for Premium. Shouldn't those users get the upsell to business?

Reply:
To avoid making unwanted changes and ensure the warning doesn't happen anymore, I rolled back most of the changes and corrected the Business plan slug.

We can follow-up with another PR to address the target upsell plan.